docs/install.md

```diff
@@ -13,8 +13,6 @@ The Flow CLI can be installed on macOS, Windows (7 or greater) and most Linux sy
 brew install flow-cli
 ```
 
-Note: it could take up to one hour for a new version to be propagated to Brew. In case you want to use the latest version immediately, follow the [instrcutions](#from-a-pre-built-binary) in the section below.
```

Comment:
Is this not relevant anymore?

Reply:
Hmm this is weird, it should be relevant and included in master, so not sure why this is visible here.